src/scripts/styles/pages/_home.scss

```diff
@@ -65,14 +65,21 @@ main {
     &:hover {
       cursor: pointer;
     }
+
+    @media (min-width: 768px) {
+      height: 77px;
+      width: 77px;
```

Comment:
[OPTIONAL] It would be better to create a mixing for this media query to keep consistency and enhance readability